torch/_C/__init__.pyi.in

```diff
@@ -161,10 +171,39 @@ class ScriptFunction:
     def qualified_name(self) -> str: ...
 
 # Defined in torch/csrc/jit/python/python_ir.cpp
-class TensorType:
+# TorchType is equivalent to torch.Type/Torch._C.Type
+class TorchType:  
```

Comment:
`TorchType` does not exist anywhere in the code base. Are you using a random name because `torch._C.Type` (which does exist) clashes with the `from typing import Type` import in this file?

Expanding the comment may be useful, I'm not quite sure what is happening here.

Reply:
Yes, that's right. I am using `TorchType` to refer to `torch._C.Type`. Not using `Type` because this would clash with `typing.Type`.

Reply:
I think the best is to rename `TorchType -> Type` and use `typing.Type` in __init__.pyi.in

Reply:
This will overlap with gh-43862 (FYI @SplitInfinity), which uses `JitType`. Seems like a good name. Maybe you can align with that PR?

Reply:
I too did not name this class `Type` in the typing stub in my PR even though it is called `Type` on the C++ side because of a conflict with `typing.Type`. I like the name `JitType` and think we should use that. I'm going to try to land my PR today.

Reply:
Okay, I will mark this PR as _draft_ and wait for gh-43862 to be merged.